jsonnet/telemeter/server/kubernetes.libsonnet

```diff
@@ -96,7 +96,7 @@ local internalPort = 8081;
         container.mixin.resources.withLimitsMixin($._config.telemeterServer.resourceLimits) +
         container.mixin.resources.withRequestsMixin($._config.telemeterServer.resourceRequests) +
         container.withVolumeMounts([tlsMount]) +
-        container.withEnv([oidcIssuer, clientSecret, clientID]) + {
+        container.withEnv([name, authorizeURL, oidcIssuer, clientSecret, clientID]) + {
```

Comment:
I think we can actually get rid of this environment variable entirely. It was previously used to give each pod a unique ID in the gossip hashring but I think it's not longer needed.

Reply:
ack, I thought so

Reply:
done